LoopKitUI/Views/Settings Editors/DeliveryLimitsReviewView.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  DeliveryLimitsReviewView.swift
+//  LoopKitUI
+//
+//  Created by Anna Quinlan on 7/6/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+import LoopKit
+import HealthKit
+
+
+public struct DeliveryLimitsReviewView: View {
+    @ObservedObject var viewModel: TherapySettingsViewModel
+    let mode: PresentationMode
+    
+    public init(mode: PresentationMode = .acceptanceFlow, viewModel: TherapySettingsViewModel){
+        self.viewModel = viewModel
+        self.mode = mode
+    }
+    
+    @ViewBuilder public var body: some View {
+        if viewModel.pumpSupportedIncrements != nil {
```

Comment:
Do you want to move this to be a `precondition` in the `init` as you've done with some of the other review views?